src/model/filter.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, scene: "Scene", filter_id: str, filter_type: int, pos=None):
         self._in_data_types: dict[str, DataType] = {}
         self._out_data_types: dict[str, DataType] = {}
 
+    def getScene(self):
```

Comment:
By the way, the property below, is the python way of implementing a getter for the scene. You just call object.scene and the getter below will be called.